convert/stl/fromStl.js

```diff
@@ -13,10 +13,7 @@ const toParser = (format) => {
   }
 };
 
-export const fromStl = async (
-  stl,
-  { format = 'ascii', geometry = 'graph' } = {}
-) => {
+export const fromStl = async (stl, { format = 'ascii' } = {}) => {
```

Comment:
What if it's binary?

Reply:
This is the default, binary can be passed in.
Probably it should check that it's one of the accepted formats.